install.sh

```diff
@@ -14,6 +11,13 @@ brew bundle
 brew unlink qt 2>/dev/null || true
 brew link --force qt5
 
+echo "Linking dotfiles into ~..."
+# Before `rcup` runs, there is no ~/.rcrc, so we must tell `rcup` where to look.
+# We need the rcrc because it tells `rcup` to ignore thousands of useless Vim
+# backup files that slow it down significantly.
+RCRC=rcrc rcup -v
```

Comment:
Dope. I'm [doing all of that manually right now](https://github.com/gfontenot/dotfiles/blob/master/setup/link-dotfiles#L8). I like this much better.